### ISSUE TITLE
Slider - add support for aria-describedby / aria-labelled by

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#7164]
**Is this a new component or an extension of an existing one?**
Slider

**Describe the feature**
We want to use the slider component in the context of a form. Similar to other form inputs, there are situations where we need to display messaging to the user about options they selected, and link the message to the input using `aria-describedby`. This is not currently supported in the current version of the Slider component. I expected that I could define `<Slider aria-describedby="xxx" ... /> and have that attribute passed down to the div with `role="slider"`.

While looking into this, I also noticed that `aria-labelledby` also does not appear to be supported for the slider component. This could be a nice enhancement, too, to provide a choice to reuse existing text in the UI.

**Are there visuals for this feature? If applicable, please include examples for each state and for varying widths**
none

**Any other information?**

**Comment by ajaypratap003** [comment]
One more issue for slider. Slider doesn't work with non-linear data.
https://github.com/patternfly/patternfly-react/issues/7180
